dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_document_path
+
+
+def upload_file(bucket, dry_run, file_path, framework_slug, bucket_category,
+                document_category, document_type):
+    supplier_id = get_supplier_id_from_framework_file_path(file_path)
+    document_name = '{0}.{1}'.format(document_category, document_type)
+    upload_path = get_document_path(framework_slug, supplier_id, bucket_category,
+                                    document_name)
+    if not dry_run:
+        with open(file_path) as file_contents:
+            bucket.save(upload_path, file_contents, acl='private', move_prefix=None)
+        print(supplier_id)
+    else:
+        print("[Dry-run] UPLOAD: '{}' to '{}'".format(file_path, upload_path))
+
+
+def get_bucket_name(stage, bucket_category):
+    bucket_name = 'digitalmarketplace-{0}-{1}-{1}'.format(bucket_category, stage)
+    print("BUCKET: {}".format(bucket_name))
+    return bucket_name
+
+
+def get_all_files_of_type(local_directory, file_type):
+    for root, subfolder, files in os.walk(local_directory):
+        for filename in files:
+            if filename.endswith(file_type):
+                yield os.path.join(root, filename)
+
+
+def get_supplier_id_from_framework_file_path(path):
+    match = re.search(r'-(\d{5,6})', path)
```

Comment:
With the new filename format, uploaded filenames should start with supplier_id, so if we're not modifying filenames generated by the pdf creation script then we should be looking for `(\d{5,6})-` instead.